src/ActivitiesSDKAdapter.js

```diff
@@ -20,21 +21,51 @@ import logger from './logger';
  * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/ActivitiesAdapter.js#L6}
  */
 
+/**
+ * Extracts JSON card and files from server activity.
+ *
+ * @private
+ * @param {object} sdkActivity  SDK activity object
+ * @returns {Array} Array of attachments.
+ */
+function parseSDKAttachments(sdkActivity) {
+  const {attachments = []} = sdkActivity;
+
+  if (sdkActivity.card) {
+    attachments.push(sdkActivity.card);
+  }
+
+  try {
+    if (sdkActivity.object && sdkActivity.object.cards) {
+      sdkActivity.object.cards.forEach((c) => {
+        const card = JSON.parse(c);
+
+        attachments.push(card);
+      });
+    }
+  } catch (err) {
+    logger.error('ACTIVITY', undefined, 'parseSDKAttachments()', `Unable parse attachments for activity with id "${sdkActivity.id}"`, err);
```

Comment:
If you're handling the error, no need to report it as an error unless you're throwing it. I would make this a warning and specify in the log the error is being handlded

Reply:
Done